amazon/ion/ioncmodule.c

```diff
@@ -116,8 +132,8 @@ static PyTypeObject ionc_read_IteratorType = {
  *  Returns:
  *      An attribute as an int
  */
-static int int_attr_by_name(PyObject* obj, char* attr_name) {
-    PyObject* py_int = PyObject_GetAttrString(obj, attr_name);
+static int int_attr_by_name(PyObject* obj, int* attr_name) {
```

Comment:
Why is this `int*` and not `PyObject*`?

Reply:
> Why is this `int*` and not `PyObject*`?

Thanks for catching this. It should be `PyObject*`, I was following the IDE recommended data type when I was editing this method.